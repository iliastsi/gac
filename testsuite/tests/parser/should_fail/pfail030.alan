hello() : proc
  foo : int;
{
  = 3;
}
