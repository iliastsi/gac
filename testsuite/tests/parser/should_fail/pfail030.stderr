pfail030.alan:4:3: Error: Parse error on input `='
	Not a valid statement

